Examples/AutomateTest/README.md

```diff
@@ -0,0 +1,12 @@
+The goal of this script is to automate the testing of react-native, react-native-code-push and their internal compatibility. Run **./run.sh** to create an app and test it. 
```

Comment:
You might be able to remove the gitignore as well, assuming that the test app can be generated in any directory